Release process is broken.

NPM can't set version.
```
npm ERR! Git working directory not clean.

npm ERR! A complete log of this run can be found in: 
npm ERR!     /home/vsts/.npm/_logs/2022-05-19T01_25_21_416Z-debug-0.log
```
```
If run in a git repo, it will also create a version commit and tag. This behavior is 
controlled by git-tag-version (see below), and can be disabled on the command line 
by running npm --no-git-tag-version version. It will fail if the working directory is 
not clean, unless the -f or --force flag is set.
```

--- a/{{ cookiecutter.project_name }}/release.config.js
+++ b/{{ cookiecutter.project_name }}/release.config.js
@@ -21,7 +21,10 @@ module.exports = {
       "@semantic-release/exec",
       {
         prepareCmd:
-          "./scripts/lint && poetry version ${nextRelease.version} && npm version ${nextRelease.version} && poetry build",
+          "./scripts/lint && " +
+          "poetry version ${nextRelease.version} && " +
+          "npm version --no-git-tag-version ${nextRelease.version} && " +
+          "poetry build",
         publishCmd: "poetry publish",
       },
     ],

